app/views/notification_mailer/meeting_reminder.text.erb

```diff
@@ -0,0 +1,4 @@
+Hi <%= @member.name %>,
+
+Here's a friendly reminder that your meeting tomorrow, "<%= @meeting.name%>", will be taking place at <%= @meeting.time %> at 
```

Comment:
Local storage sounds like it would work for most cases, but I'm not sure if it would work for reminder-type emails since they are triggered by the scheduler and there is not actually a user session in context.

I think for these types of emails you may need to store the user's preferred locale somewhere persistent that the scheduler has access to -- maybe on the `User` model.  

Then the jobs could do something like this:

```
meetings_tomorrow.each do |meeting|
  meeting.members.each do |member|
    I18n.with_locale(member.locale) do
      NotificationMailer.meeting_reminder(meeting, member).deliver_now
    end
  end
end
```

I've never actually had to make internationalized mailers before, though, so take this with a grain of salt haha.